Quadriga/src/main/webapp/WEB-INF/views/navigation/nav-settings.jsp

```diff
@@ -12,19 +12,23 @@
 <div>
     <ul>
         <li><a
-            href="${pageContext.servletContext.contextPath}/auth/workbench/projects/${project.projectId}"><span
+            href="${pageContext.servletContext.contextPath}/auth/workbench/projects/${publicpageprojectid}"><span
```

Comment:
let's use the old version, this will require every controller backing a settings page to put this variable in the model